pkg/daemon/daemon.go

```diff
@@ -206,6 +244,71 @@ func (dn *Daemon) Run(stopCh <-chan struct{}, exitCh <-chan error) error {
 	}
 }
 
+func (dn *Daemon) runWorker() {
+	for dn.processNextWorkItem() {
+	}
+}
+
+func (dn *Daemon) enqueueFoo(obj interface{}) {
```

Comment:
Maybe use a descriptive name, such as enqueueNodeState. Foo is like an example.

Reply:
done